app/shared/src/commonMain/kotlin/ui/settings/tabs/app/AppSettingsTab.kt

```diff
@@ -420,6 +421,17 @@ private fun SettingsScope.PlayerGroup(
             title = { Text("选择数据源后自动关闭弹窗") },
             Modifier.placeholder(vm.uiSettings.loading),
         )
+        if (Platform.currentPlatform.isMobile()) {
```

Comment:
```suggestion
        if (currentPlatform.isMobile()) {
```

`Platform.currentPlatform` is deprecated